client/src/main/java/sbt/client/NativeClient.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt.client;
+
+import sbt.internal.client.SimpleClient;
+import java.nio.file.Paths;
+import org.fusesource.jansi.AnsiConsole;
+
+public class NativeClient {
```

Comment:
Duplicate of `sbt.client.Client`?

Reply:
Ah, the `false` vs `true` for `useJNI`.

_mildly wonders off wondering how they could be dedupped..._

Reply:
They probably could be by adding a `--jna` flag or something. The universal script that uses java rather than graal could pass that flag in while the default is to use jni. It is nice that `sbtc` as generated by `native-image` does not need to be wrapped in a shell script to add additional arguments.

Reply:
Yep. I added a `--jna` flag and deleted `NativeClient`. Default is to use jni but the generated client script prepends `--jna` to the argument list.